data/sam_20161031_custom_specification.json

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
Did you make this manually? Is it convertible from something else?

Reply:
Unfortunatelly yes, it is an extension of the SAM specification due to the fact that certain properties (e.g. `AutoPublishAlias` of `AWS::Serverless::Function`) are missing from the JSON schema file but are present in the documentation; some properties such as `MethodSettings` of `AWS::Serverless::Api` accept as value a CFN property-type, in this case `AWS::ApiGateway::Deployment.MethodSetting`, which for the sake of convenience I've chosen to simply clone as `AWS::Serverless::Api.MethodSetting` and therefore not alter the current way `cfn-lint` looks up property types though, perhaps it might be cleaner but certainly more convoluted to do the opposite... if you have any idea on how to tackle this or perhaps even avoid it completely then please feel free to share :)